tools/perspective-build/package.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "@finos/perspective-build",
+    "version": "1.0.1",
+    "description": "Workspace-wide scripts for Perspective build, mainly esbuild and webpack configuration",
+    "private": true,
```

Comment:
Is it worth making public, I have other esbuild + perspective projects I wouldn't mind using these plugins in.

Reply:
I will add a `@finos/perspective-esbuild-plugin` package for this, but it may not be in this release.  I really want this change to be backwards compatible at least for the next release, as there is little user-facing value in this PR.  As a result, there is _a lot_ of complexity here that ultimately is not necessary, e.g. support for non-module `<script>` imports of the UMD/CDN versions.  I would prefer if this complexity is excised before inviting developers to depend on it in their own builds.

Reply:
yep sounds good